nupic/embodied/envs/real_robots/wrappers.py

```diff
@@ -36,42 +35,43 @@ class WrapRobot(object):
 
     crop_obs: Specifies whether the retina image should be cropped from
              240x320x3 to 180x180x3 (mainly cuts off white space)
-    '''
-    def __init__(self, env,crop_obs=False):
+    """
+
+    def __init__(self, env, crop_obs=False):
         self._env = env
         self.crop_obs = crop_obs
 
-        self._env.action_space = self._env.action_space['joint_command']
+        self._env.action_space = self._env.action_space["joint_command"]
         if self.crop_obs:
-            self._env.observation_space = spaces.Box(low=0, high=255.0,shape=(180,180,3), dtype='float32')
+            self._env.observation_space = spaces.Box(
+                low=0, high=255.0, shape=(180, 180, 3), dtype="float32"
+            )
         else:
-            self._env.observation_space = self._env.observation_space['retina']
+            self._env.observation_space = self._env.observation_space["retina"]
 
     def __getattr__(self, name):
         return getattr(self._env, name)
 
     def step(self, action):
-        action = {
```

Comment:
Same here, autoformatted with black. Since it fits a single line it reformatted to be a single line instead. In this situation, I personally prefer the way it was before black reformatted. 

Black is very opinionated and it is common that it doesn't make good decisions, so you have to weight issues/benefits and decide when to use it. The way I see it is that the good thing is that it removes the burden of having to make these decisions, but the bad thing is that the code doesn't always looks like you would want it look.